src/diffusers/models/modeling_utils.py

```diff
@@ -214,8 +216,14 @@ def enable_xformers_memory_efficient_attention(self):
 
         Warning: When Memory Efficient Attention and Sliced attention are both enabled, the Memory Efficient Attention
         is used.
+
+        Parameters:
+            attention_op (`Callable`, *optional*):
+                Override the default `None` operator for use as `op` argument to the
+                [`memory_efficient_attention()`](https://facebookresearch.github.io/xformers/components/ops.html#xformers.ops.memory_efficient_attention)
+                function of xFormers.
```

Comment:
```suggestion
                function of xFormers.
                
        ```suggestion
                function of xFormers.
                
        Examples:
        ```py
        from diffusers import UNet2DConditionModel
        from xformers import ...  # some attention op
        
        model = UNet2DConditionModel.from_pretrained("CompVis/stable-diffusion-v1-4", subfolder="unet")
        model.enable_xformers_memory_efficient_attention(attention_op=...)
        ```
```

Reply:
One example here would be nice